src/modes/edit_message.cc

```diff
@@ -51,6 +51,26 @@ namespace Astroid {
     edit_when_ready ();
   }
 
+  EditMessage::EditMessage (MainWindow * mw, ustring _to, ustring _cc, ustring _bcc, ustring _from) :
```

Comment:
I think we should rather extend the signature with the from = "" argument, feel free to re-org arguments and make sure to grep for other places it is used.